hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/response/key/OMKeyRenameResponseWithFSO.java

```diff
@@ -91,20 +91,19 @@ public void addToDBBatch(OMMetadataManager omMetadataManager,
           .deleteWithBatch(batchOperation, getFromKeyName());
       omMetadataManager.getKeyTable(getBucketLayout())
           .putWithBatch(batchOperation, getToKeyName(), getRenameKeyInfo());
+    }
```

Comment:
good catch

can we add a line in the PR description to say this bug is fixed?

Reply:
I don't think this is a bug. Previously we only kept snapshot checks in the `else` part as we only tracked keys. Now I moved it outside as we track both keys and directory with this PR.

Reply:
got it. makes sense. thx Aswin